src/calculateRentalCost.js

```diff
@@ -27,8 +27,25 @@
  * @return {number}
  */
 
+calculateRentalCost(7);
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40;
+  let price = 0;
+
+  for (let i = 0; i < days; i++) {
+    price = price + rentalCost;
+  }
+
+  if (days >= 3 && days < 7) {
+    price = price - 20;
+  }
```

Comment:
```suggestion
  if (days >= 3 && days < 7) {
    return rentalPrice - 20;
  }
```
You can just return the full price, no need to store it